### PR TITLE
Added amin

### LINKED ISSUE #17547: amin

### PR DESCRIPTION
**Pull request by AkshayVenkataraman**
Close #17547

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -1136,3 +1136,46 @@ def test_paddle_subtract(
...
     )
+
+
+# amin
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="amin",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=st.one_of(helpers.get_dtypes("float", "int")),
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1147]: you can put `numeric` in the `get_dtypes` method and add `complex` as unsupported dtypes if they are not supported

**Review by sherry30 — CHANGES_REQUESTED** [review]
small change requested,
Can you also fix the lint errors.

Thanks

**Comment by MuhammadNizamani** [comment]
@sherry30 @MuhammadSaeedBatikh  function `amin` has not been implemented in paddle back-end so before merging this PR, someone need to implement in `amin`  in paddle back-end.

**Comment by MuhammadSaeedBatikh** [comment]
Thanks @MuhammadNizamani for pointing this out.

**Pull request merged**